enclave-tls/src/tls_wrappers/wolfssl/un_negotiate.c

```diff
@@ -193,9 +193,9 @@ static crypto_wrapper_err_t calc_pubkey_hash(DecodedCert *crt,
 }
 
 #ifdef WOLFSSL_SGX_WRAPPER
-int verify_certificate(uint8_t *der_cert, uint32_t der_cert_len)
+int verify_certificate(void *userCtx, uint8_t *der_cert, uint32_t der_cert_len)
 {
-	tls_wrapper_ctx_t *tls_ctx;
+	tls_wrapper_ctx_t *tls_ctx = userCtx;
 #else
 int verify_certificate(int preverify, WOLFSSL_X509_STORE_CTX *store)
```

Comment:
Is there way to unify the prototype of verify_certificate() in another commit?

Reply:
I will check, if it can be unified, I will change

Reply:
if unifying the prototype, then WOLFSSL_X509_STORE_CTX structure will exposed to unstrust part completely.

Reply:
OK